backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -312,7 +312,19 @@ CREATE TABLE bakers(
     -- Fraction of finalization rewards charged by the pool owner.
     -- Stored as a fraction of an amount with a precision of `1/100_000`.
     finalization_commission
+        BIGINT,
+    -- Transaction used for self suspending.
+    -- This is not null only when a baker have send the transaction for self suspending.
+    -- This should always be null, when `inactive_suspended` column is not null.
+    self_suspended
+        BIGINT
+        REFERENCES transactions,
+    -- Block which suspended this baker due to inactivity.
+    -- This is not null only when a baker got suspended due to inactivity.
+    -- This should always be null, when `self_suspended` column is not null.
+    inactive_suspended
```

Comment:
From a user's perspective: capturing the `prime_for_suspension` event and displaying it at the front-end, so people have 24 hours to fix their node, might be good to add here similar to this final `inactive-suspension` state (I assume that is for next PR).

Reply:
Right, I'll add this and the missing indexing, but leave out the API part of this